Initializing Xeet
initializing main step 0
Starting run
>>>>>>> Starting test '003_fail_over_output' <<<<<<<
setting up output directory '/home/omer/devel/xeet/tests/testbed/xeet.out/003_fail_over_output'
Starting main phase run, 1 step(s)
Main step #0 (exec) - staring run
Running command (shell: False):
python /home/omer/devel/xeet/tests/testbed/../../scripts/testing/echo.py 'Hello World!'
---------- output start ---------
Hello World!
----------- output end ----------
Command finished with return code 0
Return code is valid
stdout differs from expected
--- /home/omer/devel/xeet/tests/testbed/xeet.out/003_fail_over_output/stp0/stdout
+++ expected_stdout
@@ -1,2 +1,2 @@
-Hello World!
+Goodbye World!
 
Main step #0 (exec) - run ended (completed, failed, duration: XXXs)
Main phase ended
Test '003_fail_over_output' ended. (status: TestPrimaryStatus.Failed, duration: XXXs)
